d2cli/main.go

```diff
@@ -87,6 +87,7 @@ func Run(ctx context.Context, ms *xmain.State) (err error) {
 	if err != nil {
 		return err
 	}
+	darkThemeClassFlag := ms.Opts.String("D2_DARK_THEME_CLASS", "dark-theme-class", "", "", "the CSS class to enable dark mode. When left unset prefers-color-scheme media query is used. See https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme.")
```

Comment:
Can you include this in the man page as well? https://github.com/terrastruct/d2/blob/master/ci/release/template/man/d2.1